.github/workflows/ci.yaml

```diff
@@ -124,6 +137,7 @@ jobs:
 
       - name: Install Deps and Build
         id: validation
+        continue-on-error: true
```

Comment:
what is the motivation for, and what are the implications of this?

Reply:
This allows the next step, sending the failure email, to execute if this step fails. There's an explicit check in the step after the email is sent that fails the job if the validation step failed.